test/fixtures/discovery/ha_cluster_discovery_ascs_ers_simple_mount.json

```diff
@@ -0,0 +1,672 @@
+{
+  "agent_id": "4b30a6af-4b52-5bda-bccb-f2248a12c992",
+  "discovery_type": "ha_cluster_discovery",
+  "payload": {
+    "DC": false,
+    "Provider": "azure",
+    "Id": "8bca366a6cb7816555538092a1ddd5aa",
+    "Cib": {
+      "Configuration": {
+        "Nodes": [
+          {
+            "Id": "1",
+            "Uname": "vmnwprd01",
+            "InstanceAttributes": null
+          },
+          {
+            "Id": "2",
+            "Uname": "vmnwprd02",
+            "InstanceAttributes": null
+          }
+        ],
+        "CrmConfig": {
+          "ClusterProperties": [
+            {
+              "Id": "cib-bootstrap-options-have-watchdog",
+              "Name": "have-watchdog",
+              "Value": "true"
+            },
+            {
+              "Id": "cib-bootstrap-options-dc-version",
+              "Name": "dc-version",
+              "Value": "2.0.5+20201202.ba59be712-4.13.1-2.0.5+20201202.ba59be712"
+            },
+            {
+              "Id": "cib-bootstrap-options-cluster-infrastructure",
+              "Name": "cluster-infrastructure",
+              "Value": "corosync"
+            },
+            {
+              "Id": "cib-bootstrap-options-cluster-name",
+              "Name": "cluster-name",
+              "Value": "netweaver_cluster"
+            },
+            {
+              "Id": "cib-bootstrap-options-stonith-enabled",
+              "Name": "stonith-enabled",
+              "Value": "true"
+            },
+            {
+              "Id": "cib-bootstrap-options-stonith-timeout",
+              "Name": "stonith-timeout",
+              "Value": "144s"
+            },
+            {
+              "Id": "cib-bootstrap-options-maintenance-mode",
+              "Name": "maintenance-mode",
+              "Value": "false"
+            }
+          ]
+        },
+        "Resources": {
+          "Clones": null,
+          "Groups": [
+            {
+              "Id": "grp_NWP_ASCS00",
+              "Primitives": [
+                {
+                  "Id": "rsc_ip_NWP_ASCS00",
+                  "Type": "IPaddr2",
+                  "Class": "ocf",
+                  "Provider": "heartbeat",
+                  "Operations": [
+                    {
+                      "Id": "rsc_ip_NWP_ASCS00-monitor-10s",
+                      "Name": "monitor",
+                      "Role": "",
+                      "Timeout": "20s",
+                      "Interval": "10s"
+                    }
+                  ],
+                  "MetaAttributes": null,
+                  "InstanceAttributes": [
+                    {
+                      "Id": "rsc_ip_NWP_ASCS00-instance_attributes-ip",
+                      "Name": "ip",
+                      "Value": "10.80.1.25"
+                    }
+                  ]
+                },
+                {
+                  "Id": "rsc_SAPStartSrv_NWP_ASCS00",
+                  "Type": "SAPStartSrv",
```

Comment:
Here we use a `SAPStartSrv` resource instead of a `Filesystem` resource

Reply:
SAP is fun!